src/CheckoutSdk/Common/CardholderAccountAgeIndicatorType.cs

```diff
@@ -0,0 +1,20 @@
+using System.Runtime.Serialization;
+
+namespace Checkout.Common
+{
+    public enum CardholderAccountAgeIndicatorType
+    {
+        [EnumMember(Value = "less_than_thirty_days")]
+        LessThanThirtyDays,
+
+        [EnumMember(Value = "more_than_sixty_days")]
+        MoreThanSixtyDays,
+        [EnumMember(Value = "no_account")] NoAccount,
```

Comment:
Is this a correct enumerator?

Reply:
I have attached here: https://api-reference.checkout.com/#operation/createSession!path=account_info/account_password_change_indicator&t=request. I will format it correctly